sma-webconnect.js

```diff
@@ -33,6 +33,28 @@ module.exports = function (RED) {
     }
   };
 
+  const sb = {
+    "id": "1",
+    "values": {
+      "6100_00464800": {
+        "name": "phase1_voltage",
+        "divider": 100
+      },
+      "6100_00465700": {
+        "name": "frequency",
+        "divider": 100
+      },
+      "6100_40465300": {
+        "name": "phase1_current",
+        "divider": 1000
+      },
+      "6100_0046C200": {
+        "name": "phase1_power",
+        "divider": 1
+      }
+    }
+  };
+
```

Comment:
I'm wondering if it would be cleaner to skip the "phase1_" prefixes here, as from my understanding those Sunny Boys only have one phase.

Reply:
Yes, you are correct that the Sunny Boys are single phase, however in the GUI the three phases are listed.
![image](https://user-images.githubusercontent.com/3389734/107873150-1e104d00-6f00-11eb-8019-70df9c9424ab.png)
I am happy to go either way with the naming.

Reply:
On closer inspection the power value does not have a "phase" prefix so I will change it.

Reply:
Okay, then I think I'd be okay with keeping the "phase" prefix for the voltage. 👍